web/src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
 import React, { useEffect, useState } from 'react'
-import Plot from 'react-plotly.js'
+import Plot, { PlotParams } from 'react-plotly.js'
```

Comment:
```suggestion
import Plot from 'react-plotly.js'
```

Reply:
SonarCloud is complaining that PlotParams is never used. But then I don't understand how linting passed...